github/github.go

```diff
@@ -562,7 +562,6 @@ func (c *Client) Do(ctx context.Context, req *http.Request, v interface{}) (*Res
 			}
 
 			aerr.Raw = b
-			return response, aerr
```

Comment:
Sorry, this change is not correct. Line 565 returns `aerr` and line 568 returns `err`.

Reply:
Thank you for your comment.

I think both of `aerr` and `err` are pointing to the same error struct. The `if` statement in the line `557` does assertion if the error type is `*AcceptedError`, if so, it copies the response body to the error without replacing it with another error. So, I think they are the same.

Do I miss something?

Reply:
Ah, I see what you are saying.  Yes, I think you are correct.
Let's see what @gauntface thinks about these changes.